policies/codingstyle-updates.txt

```diff
@@ -0,0 +1,75 @@
+A summary of the discussion thread so far.  Not surprisingly, it's all about
+the whitespace. :)
+
+The descriptions here were written to be understandable stand-alone.  Once we
+come to a conclusion, we'll wordsmith them into the coding style.
+
+Do not put a space after sizeof; do use parens:
+
+	sizeof(foo)
+
+When breaking long lines, if there are Boolean conditionals, put them at the
+start of the line.  Consider doing this consistently, and don't merge even if
+they fit.  Here is a bad example, it should be three lines:
+
+	some_long_condition()
+	&& short1() && short2()
+
+Related conditions should be on one line if they fit, else given an extra
+level of indent
+
+	some_long_condition()
+	    && (short1() || short2())
+	    && some_other_flag != 0

```

Comment:
I don't quite follow why extra indent here. I mean how is it different from above example? If short1() and short2()  are strongly related and deserve to be in (), then how is it different from just short1()?

Reply:
Oh! On related note I wonder if it makes sense to explicitly recommend following formatting for multi-line `?:`
```
condition ? this()
          : that();
```
In other words with `:` aligned with `?`

Reply:
The nested indent examples were wrong, I fixed them.
I am curious what others think of ?:  I like this:
```
foo()
    ? that()
    : this()
```
for what it's worth.

Reply:
Please, when coming up with this, consider creating cc-mode mods for us emacs users.
(otherwise, I think I know whose name I'm gonna scream in frustration ;-))